### DISCUSSION TITLE
EMQX v4.4 authen with salt, bcrypt error

### DISCUSSION THREAD

**Discussion by ghost** [emqx/emqx#10507] [Q&A]
Hi everyone, my english is so bad
When i connect >100 client to emqx server with authen salt, bcrypt so 1 core in cpu full 100% and emqx server drop all connection.
Then client can't connect, sub, pub topic to emqx server. Please help me!
I have 3 nodes emqx in cluster (8gb ram | 4 core cpu)
![Screenshot 2023-04-25 at 11 21 53](https://user-images.githubusercontent.com/66189660/234173786-3eaa8a10-de55-4ba5-aa09-70d8373b58b3.png)

**Comment by qzhuyan** [comment]
Pls open one github issue and fill in the related info in the template.
